### ISSUE TITLE
Invalid syntax

### ISSUE THREAD

**Issue by MS-design-ui** [BreadedChickenator/Cross-Seed-AutoDL#2]
Running Ubuntu 18 I get this after configuration

`File "CrossSeedBDs.py", line 77
    editedTitle = re.sub(AKA_DUAL_LANG_NAME_RE, r'\1', title, flags=re.IGNORECASE) + f' {year} {group}'
                                                                                                      ^
SyntaxError: invalid syntax`

**Comment by MS-design-ui** [comment]
Hard to see monospace font. Here's a screenshot: https://prnt.sc/syd23x

**Comment by BC44** [comment]
You're probably running this as python2. Run this with python3

**Comment by MS-design-ui** [comment]
```
Searching for 1 of X: ['TITLE']      ['FOLDER', 44543050712]

Traceback (most recent call last):
  File "CrossSeedBDs.py", line 211, in <module>
    main()
  File "CrossSeedBDs.py", line 96, in main
    returnedJSON = json.loads(source)
  File "/usr/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

```

Is there a requirements.txt I need to install with pip3? (if there are any)

Python 3.7.5 is the version I have installed too.

**Comment by BC44** [comment]
Only requests needs to be installed. Did you input your jackett API key correctly?

**Comment by MS-design-ui** [comment]
Can confirm requests is installed. 

Jackett API is correctly selected. I am using Jackett on a remote server and I have the domain put in there. For the port I have put 443 (SSL with Nginx).

I have tried removing the port altogether and got an error saying it's not defined. I then proceeded to leave it blank and got the same message as the one above (json,decoder...)

**Comment by MS-design-ui** [comment]
EDIT: I have fixed it by pointing it to IP rather than Nginx. I now have an issue with it saving the .torrent

`>> Found possible match. Downloading

Traceback (most recent call last):
  File "CrossSeedBDs.py", line 211, in <module>
    main()
  File "CrossSeedBDs.py", line 102, in main
    found, announceDownloadings = findMatchingTorrent(returnedJSON, pathSize, listingPath)
  File "CrossSeedBDs.py", line 194, in findMatchingTorrent
    downloadTorrent(downloadURL, listingTitle)
  File "CrossSeedBDs.py", line 206, in downloadTorrent
    with open(downloadPath, 'wb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/HIDDEN/torrents/BDs/100.Tears.2007.COMPLETE.BLURAY [1080p / Blu-ray / BD50 / m2ts / Checked / Director&#39;s Cut].torrent'`


I have made sure that the folder path exists

**Comment by BC44** [comment]
It seems like I forgot to sanitize the paths for linux. The torrent name contains forward slashes which need to be sanitized. I'll Push an update shortly.

**Comment by MS-design-ui** [comment]
Perfect. Thanks <3

**Comment by BC44** [comment]
Issue fixed in 109b1cb5d482c63d9f5835b30accaad73266f5b7

**Comment by MS-design-ui** [comment]
Confirm this works now. Closing the ticket! THanks a lot!
